web-ng/src/app/shared/models/form/field.model.ts

```diff
@@ -48,4 +49,21 @@ export class Field {
     readonly required: boolean,
     readonly multipleChoice?: MultipleChoice
   ) {}
+
+  /**
+   * Returns an option but only if the given field is of a proper type (MULTIPLE_CHOICE).
```

Comment:
Nit: of a proper type ->  For MULTIPLE_CHOICE fields, returns the option with the specified id. Throws an Error if called for other field types.

Also, should we add a TODO to create subclasses of Field for each field type?